e2e-tests/specs/relay-general-functionality.spec.ts

```diff
@@ -15,26 +15,16 @@ test.describe('Free - General Functionalities, Desktop', () => {
     test.slow()
     await expect(async () => {
       await dashboardPage.generateMask(5)
-      expect(await page.locator(dashboardPage.maskCard).count() === 5)
+      expect(await page.locator(dashboardPage.maskCardString).count() === 5)
     }).toPass()
 
 
     // After five times, the button becomes greyed-out and the user cannot add other masks anymore (TODO: for a free user from a country where Premium is NOT available).
     expect(await dashboardPage.maxMaskLimitButton.textContent()).toContain('Get unlimited email masks')
   })
-
-  test('Check that when generating a new mask, its card is automatically opened, C1686210, C1553075, C1553064', async ({ dashboardPage, page }) => {
```

Comment:
Why the deleted test?

Reply:
validated in other tests, that need to interact with the cards